src/Change.php

```diff
@@ -663,6 +663,15 @@ public static function getProcessStatusArray()
         return $tab;
     }
 
+    /**
+     * Get the ITIL object reopenable status list
+     *
+     * @return array
+     */
```

Comment:
No need to duplicate the docblock when it overrides a method. Indeed, almost all IDEs or tools related to code documentation are automatically showing the parent docblock in this case.
Having only one place to change when docblock has to be updated is preferable.
```suggestion
```